### ISSUE TITLE
[LOC] "される1" needs to be replaced with "1","される2" needs to be replaced with "2"

### ISSUE THREAD

**Issue by hyoshioka0128** [MicrosoftDocs/windows-itpro-docs#3885]
https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/microsoft-edge-kiosk-mode-deploy
![microsoft-edge-kiosk-mode-deploy_j](https://user-images.githubusercontent.com/40815708/58750499-5a064300-84ce-11e9-946f-5f5efd34e46f.png)

---
#### ドキュメントの詳細

⚠ *このセクションを編集しないでください。 docs.microsoft.com で必須です ➟ GitHub の問題のリンク。*

* ID: f840fa14-a379-d297-c3f7-ada05ec299d6
* Version Independent ID: 81396f7b-7506-43cd-fd9a-4066719a6eb2
* Content: [Microsoft Edge キオスク モードの展開](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/microsoft-edge-kiosk-mode-deploy)
* Content Source: [browsers/edge/microsoft-edge-kiosk-mode-deploy.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/browsers/edge/microsoft-edge-kiosk-mode-deploy.md)
* Product: **edge**
* Technology: **microsoft-edge**
* GitHub Login: @eavena
* Microsoft Alias: **eravena**

**Comment by illfated** [comment]
This issue is most likely caused by the translation procedure reading the format tag "superscript" (`<sup>`) as text instead of code (for footnote indicators). Whereas I am fairly sure that the tag `<sup>` is the root cause, I do not have access to knowledge about the translation script process, if it translates from the web content on docs.microsoft.com/en-us/ or if it translates from the Github content. The only difference is what the translation script input looks like.

The mentioned issue also happens further down in the same table (text copied from the raw view of the Github source):
```markdown
AllowFlashClickToRun [ ... ] ![Supported](images/148767.png)<sup>2</sup>
AllowMicrosoftCompatibilityList [ ... ] ![Supported](images/148767.png)<sup>1</sup>
EnterpriseModeSiteList [ ... ] ![Supported](images/148767.png)<sup>1</sup>
SendIntranetTraffictoInternetExplorer [ ... ] ![Supported](images/148767.png)<sup>1</sup>
ShowMessageWhenOpeningInteretExplorerSites [ ... ] ![Supported](images/148767.png)<sup>1</sup>
SyncFavoritesBetweenIEAndMicrosoftEdge [ ... ] ![Supported](images/148767.png)<sup>1</sup>
```

A possible workaround could be to use the Windows / OEM ASCII code for superscript ^1  ( ` ¹ ` ) and superscript ^2 ( ` ² ` ) instead of the HTML tag `<sup>`:
Superscript 1 = ¹ (Alt+0185)
Superscript 2 = ² (Alt+0178)
Superscript 3 = ³ (Alt+0179)

From https://www.toptal.com/designers/htmlarrows/math/ :

| DEGREE SIGN        | SUPERSCRIPT ONE     | SUPERSCRIPT TWO     | SUPERSCRIPT THREE   |
| :--------------:   | :--------------:    | :--------------:    | :---------------:   |
| °                  | ¹                   | ²                   | ³                   |
| UNICODE: U+000B0   | UNICODE: U+000B9    | UNICODE: U+000B2    | UNICODE: U+000B3    |
| HEX CODE: `&#xb0;`   | HEX CODE: `&#xb9;`    | HEX CODE: `&#xb2;`    | HEX CODE: `&#xb3;`    |
| HTML CODE: `&#176;`  | HTML CODE: `&#185;`   | HTML CODE: `&#178;`   | HTML CODE: `&#179;`   |
| HTML ENTITY: `&deg;` | HTML ENTITY: `&sup1;` | HTML ENTITY: `&sup2;` | HTML ENTITY: `&sup3;` |

Trivia note: While using MS-DOS in the 1990s I picked up the habit of using Alt+253 = ² (Superscript 2) and Alt+252 = ³ (Superscript 3), but I never found a need for Superscript 1 for some reason.

**Comment by e0i** [comment]
@officedocsbot assign @e0i

**Comment by e0i** [comment]
@officedocsbot assign @dongmicro
@officedocsbot add label localization

**Comment by e0i** [comment]
@officedocsbot assign @MollyMalone

**Comment by e0i** [comment]
@officedocsbot unassign @MollyMalone
@officedocsbot unassign @TinaMcN

**Comment by e0i** [comment]
@officedocsbot unassign @dongmicro

**Comment by TinaMcN** [comment]
Thank you for the feedback.  The article was machine translated.  I sent it for human translation.  It will take up to 4 weeks to complete the translation.
フィードバックをありがとうございました。 記事は機械翻訳されました。 私は人間の翻訳のためにそれを送った。 翻訳が完了するまでに最大4週間かかります。

**Comment by TinaMcN** [comment]
@hyoshioka0128 記事は現在、人間が翻訳されています。 問題が解決したことを確認してください。 フィードバックをありがとう。
(The article is now human translated. Please verify that the issue is fixed. Thanks again for the feedback.)

**Comment by illfated** [comment]
MDM 設定 | デジタル /対話型サイネージ | パブリック ブラウジングシングル アプリ | パブリック ブラウジング複数アプリ | 通常モード
-- | :--: | :--: | :--: | :--:
AllowFlashClickToRun | ![Supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148767.png)<sup>2</sup> | ![Supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148767.png) | ![Supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148767.png) | ![Supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148767.png)
AllowFullscreen* | ![Not supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148766.png) | ![Supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148767.png) | ![Supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148767.png) | ![Supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148767.png)
AllowInPrivate | ![Not supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148766.png) | ![Not supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148766.png) | ![Not supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148766.png) | ![Supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148767.png)
AllowMicrosoftCompatibilityList | ![Not supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148766.png) | ![Not supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148766.png) | ![Supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148767.png)<sup>1</sup> | ![Supported](https://docs.microsoft.com/ja-jp/microsoft-edge/deploy/images/148767.png)

Seems OK to me, issue looks like it is resolved.

**Comment by hyoshioka0128** [comment]
@TinaMcN  @illfated  -Fixed.
Closed #3885.
